GoRest.Api.Tests/Comments/Create/CreateCommentsNegative.cs

```diff
@@ -0,0 +1,118 @@
+﻿using API_Tests.Asserts;
+using API_Tests.Helpers;
+using FluentAssertions;
+using GoRest.Api.Client.Client;
+using GoRest.Api.Client.Client.Builder;
+using GoRest.Api.Client.Client.Extentions;
+using GoRest.Api.Client.Client.Interfaces.Controllers;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+namespace API_Tests.Posts.Create.CreateCommentNegativeTests
+{
+    [Parallelizable]
```

Comment:
this attribute is not needed here.
If all tests should be parralel, [Parallelizable(All)] in AssemblyInfo is enough

Reply:
done